src/darwin/Framework/CHIP/templates/MTRBaseClusters-src.zapt

```diff
@@ -12,16 +12,248 @@
 #import "MTRCommandPayloadsObjc.h"
 
 #include <lib/support/CHIPListUtils.h>
+#include <lib/core/CHIPTLV.h>
 #include <platform/CHIPDeviceLayer.h>
 #include <type_traits>
-#include <controller/TypedReadCallback.h>
+#include <app/ReadClient.h>
 
 using chip::Callback::Callback;
 using chip::Callback::Cancelable;
 using namespace chip::app::Clusters;
 using chip::Messaging::ExchangeManager;
 using chip::SessionHandle;
 
+template <typename BridgeType, typename DecodableAttributeType>
```

Comment:
since these do not seem to depend on input data, could we place them in some stand-alone sources? I think code-gen code should be the one that actually needs codegen and non-codegen should be as regular "source"

Reply:
We can, probably.  I'll think about a sane place to put them in.

Reply:
Done.